js/ui/html_editor/modules/toolbar.js

```diff
@@ -181,6 +194,37 @@ if(Quill) {
             }
         }
 
+        _detectKeyboardFormatChanging() {
+            const namespace = addNamespace('keydown', this.editorInstance.NAME);
+
+            eventsEngine.off(this.editorInstance._getQuillContainer(), namespace);
+            eventsEngine.on(this.editorInstance._getQuillContainer(), namespace, ({ originalEvent }) => {
+                const pressedKey = originalEvent && normalizeKeyName(originalEvent);
+
+                if(pressedKey) {
+                    each(FORMAT_HOTKEYS, (index, { key, formatName }) => {
+                        if(pressedKey.toLowerCase() === key && isCommandKeyPressed(originalEvent)) {
```

Comment:
we can use Object/Map to store FORMAT_KEYS instead of list to find a necessary format with O(1) instead of O(formats count).

It'll be smth like:

```
const FORMAT_HOTKEYS_MAP = { b" 'bold' };

...

const formatName = FORMAT_HOTKEYS_MAP[pressedKey];

if (formatName) {
    this._updateButtonState(formatName);

    return false;
}
...


```